src/rbac.rs

```diff
@@ -94,17 +94,18 @@ impl Authorization {
             trace!(matches = false, "empty groups");
             return false;
         }
-        for rule in self.groups.iter() {
-            // If ANY rule matches, it is a match...
-            let mut rule_match = true;
-            for group in rule.iter() {
-                // We need ALL groups to match...
-                let mut group_match = true;
-                for mg in group.iter() {
+        for group in self.groups.iter() {
```

Comment:
**Suggestion:** This naming is consistent with the internal API we're using but rubs a bit against the public API for an AuthorizationPolicy where what we're internally referring to as a group is called a Rule. It might be helpful to add a comment noting that naming comes from `proto/authorization.proto` and is inconsistent with `v1beta1 AuthorizationPolicy`.

Could alternatively or also be added to `impl TryFrom<&XdsRbac> for Authorization`.

Reply:
Yeah I think we should actually change `authorization.proto` names to be more like AuthorizationPolicy for clarity; we can make that breaking change now since we are pre-launch

Reply:
Probably best to save breaking changes for after Kubecon EU. I'd be down to work on changes to the proto though.

Reply:
Created #500 to track proposals, decision and potentially implementation